doc/source/actors.rst

```diff
@@ -539,3 +688,7 @@ Concurrency within an actor
   .. group-tab:: Java
 
     Actor-level concurrency hasn't been implemented in Java yet.
```

Comment:
Java support is already implemented. @jovany-wang @kfstorm could you open a PR to fix this?